Editor/UI/JitInspectorView.cs

```diff
@@ -0,0 +1,330 @@
+using Iced.Intel;
+using JitInspector.Search;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEditor;
+using UnityEditor.UIElements;
+using UnityEngine;
+using UnityEngine.Profiling;
+using UnityEngine.UIElements;
+using Label = UnityEngine.UIElements.Label;
+
+namespace JitInspector.UI
+{
+    using static StringBuildingExtensions;
+
+    public class JitInspectorView : EditorWindow
+    {
+        private static SearchableMethodIndex s_methodIndex = new SearchableMethodIndex();
+        private static HashSet<string> s_hashSet = new HashSet<string>();
+        private static List<Assembly> s_assemblies;
+
+        private readonly float _delay = 0.45f;
+
+        [SerializeField]
+        private VisualTreeAsset _jitInspectorTemplateAsset = default;
```

Comment:
nit: `default` is implicit inside a `class`
```suggestion
        private VisualTreeAsset _jitInspectorTemplateAsset;
```